hydra-http-form.c

```diff
@@ -1429,6 +1439,7 @@ void usage_http_form(const char *service) {
          "The following parameters are optional:\n"
          " (c|C)=/page/uri     to define a different page to gather initial "
          "cookies from\n"
+         " (g|G)= skip pre-requests - only use this when no pre-cookies are required\n"
```

Comment:
this is missing a lot of spaces. the "skip .." test must be aligned to the other description indention. thanks.

Reply:
I fixed the intendation